modules/docs/mdx/MAINTAINING.mdx

```diff
@@ -0,0 +1,250 @@
+import {Meta} from '@storybook/addon-docs';
+
+<Meta title="Welcome/Maintaining" />
+
+# Maintaining Canvas Kit
+
+If you're a Canvas Kit core maintainer, this doc is for you! Consider this a field guide to help you
+maintain Canvas Kit with confidence. If you see some information that's unclear, incomplete, or
+incorrect, please update this doc to help your future self and others. Thanks for maintaining Canvas
+Kit!
+
+- [Branches](#branches)
+- [GitHub Actions](#github-actions)
+  - [Forward Merge Workflow](#forward-merge-workflow)
+  - [Release Minor Workflow](#release-minor-workflow)
+  - [Release Workflow](#release-workflow)
+- [Releases](#releases)
```

Comment:
One question I have: Should these be separate docs? Should we have a standalone "Releasing" doc?

Reply:
I think that having it live here is beneficial. Since releasing would only fall to this audience. Unless you think that the release doc is too large and will just create more noise than good within this doc

Reply:
Let me think about this... probably good here, but I will say contributors and consumers can be interested in how code gets shipped at the right level of detail for that audience. For example, if code is merged, when does it ship?

Reply:
Yeah, I think we could have an abbreviated section for consumers on our release cadence.

Reply:
Let's see how long this doc gets and then parse it out. I'm okay with it living here for now.